sparta/simdb/include/simdb/schema/ColumnMetaStructs.hpp

```diff
@@ -23,14 +23,20 @@ namespace simdb {
 template <typename ColumnT, typename Enable = void>
 struct column_info;
 
+template<typename T>
+struct is_pod {
+  static constexpr bool value = std::is_trivial<T>::value
+                             && std::is_standard_layout<T>::value;
+};
+
```

Comment:
Wonder if this could be added to sparta/sparta/utils/MetaStructs.hpp reused in the multiple places you've changed.

One concern I have with this definition here -- it's not in a namespace, which means if someone did something silly like `using namespace std;` and then included this file, the compiler might see multiple definitions.

Reply:
I have moved the struct to a new file in the simdb include directory. This is because it appeared that simdb files were being included in both simdb and sparta, but I didn't see the sparta path being passed as an include for the simdb compilation. The struct is now located at "simdb::utils::is_pod"